kinto/core/resource/__init__.py

```diff
@@ -474,6 +474,9 @@ def patch(self):
 
         updated = self.apply_changes(existing, changes=changes)
 
+        # remove field if null attribute is passed (RFC 7396)
+        updated = {key: value for key, value in updated.items() if value}
```

Comment:
I would use `if value is not None` because we don't want to remove it for `False` and `0`
You might to add a test to make sure we don't remove the field with `False`, `0`, `""` (empty string) etc.